### PR TITLE
feat: add JDK21 version jdk21-2023-08-09-06-56-beta prerelease

### PR DESCRIPTION
**Pull request by smerle33**
as per https://github.com/jenkins-infra/helpdesk/issues/3709

### CODE REVIEW

File: provisioning/ubuntu-provision.sh [hunk 1]
```diff
@@ -286,7 +286,7 @@ function install_jdk() {
   apt-get install --yes --no-install-recommends fontconfig
 
   ## OpenJDKs: Adoptium - https://adoptium.net/installation.html
-  mkdir -p /opt/jdk-8 /opt/jdk-11 /opt/jdk-17 /opt/jdk-19
+  mkdir -p /opt/jdk-8 /opt/jdk-11 /opt/jdk-17 /opt/jdk-19 /opt/jdk-21
```
Comment (dduportal) [inline provisioning/ubuntu-provision.sh:289]:
```suggestion
  for jdkVersion in 8 11 17 19 21; do
    echo "/opt/jdk-$jdkVersion"
  done
```

(Nit)

Comment (timja) [reply provisioning/ubuntu-provision.sh:289]: echo?

**Review by dduportal — CHANGES_REQUESTED** [review]
Don't forget the Goss tests: https://github.com/jenkins-infra/packer-images/blob/f4b1666a0cea84411d473543579a980afe94d559/goss/goss.yaml#L2-L21

**Review by lemeurherve — APPROVED** [review]
🚀

**Review by dduportal — APPROVED** [review]
LGTM

**Pull request merged**